plugin.rb

```diff
@@ -61,6 +61,7 @@ class Engine < ::Rails::Engine
     put "/discourse-reactions/posts/:post_id/custom-reactions/:reaction/toggle" => "custom_reactions#toggle", constraints: { format: :json }
     get "/discourse-reactions/posts/my-reactions" => "custom_reactions#my_reactions", as: "my_reactions"
     get "/discourse-reactions/posts/reactions-received" => "custom_reactions#reactions_received", as: "reactions_received"
+    get "/discourse-reactions/:post_id/reactions-users" => "custom_reactions#post_reactions_users", as: "post_reactions_users"
```

Comment:
Why don't we just add this on posts route ?

`/posts/:id/reactions`

Also I'm unclear why we call it "reactions users", it's reactions no ?

Reply:
Sure that's cool, ill add it to posts route
We are returning users of reactions here. so I think reactions-users sounds good. thoughts?

Reply:
I don't know I feel like we are just return reactions, and reactions have users and other stuff.... so reactions seems good enough to me no ?

Reply:
Yes, that's right we are returning reactions with users. But the reason we are creating this route was for lazy loading of reaction-users and that's where this route will be used. so naming it as per its use was what I thought. Thoughts?